tools/FakeItEasy.PrepareRelease/Program.cs

```diff
@@ -0,0 +1,280 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Runtime.CompilerServices;
+using System.Text.RegularExpressions;
+using System.Threading.Tasks;
+
+using Octokit;
+using Octokit.Helpers;
+
+namespace FakeItEasy.PrepareRelease
+{
+    static class Program
+    {
+        private const string RepoOwner = "FakeItEasy";
+        private const string RepoName = "FakeItEasy";
+        private const string ReleaseFromBranchName = "develop";
+        private const string TargetBranchName = "master";
+        private const string ExistingReleaseName = "vNext";
+        private const string NextReleaseName = ExistingReleaseName;
+
+        static async Task Main(string[] args)
+        {
+            var version = args.FirstOrDefault();
+            if (version == null)
+            {
+                throw new Exception("No version supplied");
+            }
+
+            var releaseBranchName = $"release/{version}";
+
+            var gitHubClient = GetAuthenticatedGitHubClient();

```

Comment:
This was a `static var` in the script so that it could be used by other methods. The equivalent in a real C# program would have been to make it a static field, but it felt ugly, so instead I kept it as a local variable and converted the static methods to extension methods with `gitHubClient` as the `this` parameter.